code/game/objects/items/control_wand.dm

```diff
@@ -35,7 +35,6 @@
 	if(QDELETED(data.user))
 		return
 	var/toggled = data.data["data"]
-	to_chat(data.user, "<span class='notice'>Door [toggled] toggled</span>")
```

Comment:
If this doesn't to_chat anymore, waht point is there left to this proc? It doesn't appear to do anything.

Reply:
Excellent point 👍